spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/EnableZipkinStreamServer.java

```diff
@@ -25,6 +25,16 @@
 
 import zipkin.server.EnableZipkinServer;
 
+/**
+ * Enables collecting spans via Spring Integration. spans will be received
```

Comment:
When enabled, instrumented apps will transport spans over a Spring Cloud Stream, for example RabbitMQ.

.. then details one should know.